### ISSUE TITLE
Context menu with open file/open location (especially for defect files)

### ISSUE THREAD

**Issue by rugk** [qarmin/czkawka#1114]
**Feature Description**
I would like to right-click on a file and open it with my default e.g. image viewer.
Also, maybe a "Open file location" to open the directory (in the ideal case with the file being selected by default) would be helpful.

**Background:**
Especially for the defect files search, I would like to double-check whether the file is _really_ defect or some app can still open it etc. For others, I might want to cross-check the directory context etc.

Generally though, you could make more use of the context menu there.

**Comment by yimengqinyao** [comment]
I also agree with this idea. After all, sometimes you need to navigate to Explorer to view the corresponding file. It's really inconvenient to not be able to jump directly in the software.
